openfl/federated/plan/plan.py

```diff
@@ -458,6 +459,16 @@ def get_collaborator(self, collaborator_name, root_certificate=None, private_key
 
         defaults[SETTINGS]['compression_pipeline'] = self.get_tensor_pipe()
         defaults[SETTINGS]['task_config'] = self.config.get('tasks', {})
+
+        opt_treatment = defaults[SETTINGS]['opt_treatment']
+        if isinstance(opt_treatment, str) and hasattr(OptTreatment, opt_treatment):
+            defaults[SETTINGS]['opt_treatment'] = OptTreatment[opt_treatment].value
```

Comment:
Maybe better to save string here to easeir review plan.yaml.
In `experiment` it is using `plan.config['collaborator']['settings']['opt_treatment'] = opt_treatment.name`.
We should save values in plan in the same format.

Reply:
Yes I and @igor-davidyuk discussed about it and thought of using value instead of name